src/panel/components/Pane.tsx

```diff
@@ -5,17 +5,30 @@ import React, {
   useMemo,
   MouseEventHandler,
   ComponentProps,
+  useRef,
 } from "react";
 import styled from "styled-components";
 import { useOrientationWatcher } from "../hooks";
 
-const PaneRoot: FC<ComponentProps<typeof PaneContainer>> = ({
+interface OverrideProps {
+  forcedOrientation?: { isPortrait: boolean };
+  initSize: { x: number; y: number };
+}
+
+const PaneRoot: FC<ComponentProps<typeof PaneContainer> & OverrideProps> = ({
   children,
+  forcedOrientation,
+  initSize,
```

Comment:
Something like this was needed but I didn't want to hoist the state of the panel outside of the component. This looks to work great!